File: Competencias/TopTEENProgrammer/gemelos.py
#tomaras el numero minio de monedas, cuya suma de valores es estrictamente mayor que la suma de los valores de las monedas restantes


n = int(raw_input())
money=list(map(int,raw_input().split()))
c = 0
acum = 0
suma = 0
nacum =0
money.sort(reverse=True)
for i in range(n):
    acum+=money[i]
for j in range(n):
    suma += money[j]
    nacum = acum - suma
    if suma > nacum:
        print(j+1)
        break

File: Competencias/TopTEENProgrammer/gemelos.cpp
#include <iostream>
#include <bits/stdc++.h>

using namespace std;

int main() {
    int f, c, acum = 0, suma = 0, nacum;
    cin >> f;
    int arr[f] = {}; 
    int n = sizeof(arr)/sizeof(arr[0]); 
    for (int i = 0; i < f; ++i) { 
        cin >> c;
        arr[i] = c;
    }
    sort(arr, arr+n, greater<int>());

    
    for (int i = 0; i < n; ++i) {
        acum += arr[i];
    }
    for(int j=0; j<f; j++){
            suma += arr[j];
            nacum = acum - suma;
            if(suma>nacum) {
                cout << j+1;
                return 0;
            }
    }
    return 0;
}